tools/testbench/ll_schedule.c

```diff
@@ -12,10 +12,6 @@ int schedule_task_init_ll(struct task *task, uint32_t uid, uint16_t type,
 			  enum task_state (*run)(void *data), void *data,
 			  uint16_t core, uint32_t flags)
 {
-	int ret;
```

Comment:
Lets use a prefix for commit subject e.g: ```testbenc: Remove uneeded variable```

Reply:
Okay, did it.

Reply:
@PayalKshirsagar I was referring to the commit message not the Pull Request. Just use git commit --amend in your setup, modify the commit and then use --force to push the branch. (git push -f )